quarkus-conference/web-app/.env.production

```
@@ -1,4 +1,4 @@
-REACT_APP_SESSION_SERVICE="http://microservice-session:8081"
-REACT_APP_SPEAKER_SERVICE="http://microservice-speaker:8082"
-REACT_APP_SCHEDULE_SERVICE="http://microservice-schedcule:8083"
-REACT_APP_VOTE_SERVICE="http://microservice-vote:8084"
\ No newline at end of file
+REACT_APP_SESSION_SERVICE="http://microservice-session:8080"
+REACT_APP_SPEAKER_SERVICE="http://microservice-speaker:8080"
+REACT_APP_SCHEDULE_SERVICE="http://microservice-schedcule:8080"
```

```suggestion
REACT_APP_SCHEDULE_SERVICE="http://microservice-schedule:8080"
```

I haven't checked the app or the resulting deployment, but just fixing a typo.

---

Good catch!
Anyway, this file can not be used anymore, so the typo is not impacting :'(